torch/nn/modules/linear.py

```diff
@@ -178,4 +179,62 @@ def extra_repr(self) -> str:
             self.in1_features, self.in2_features, self.out_features, self.bias is not None
         )
 
+
+class LazyLinear(LazyModuleMixin, Linear):
+    r"""A :class:`torch.nn.Linear` module with lazy initialization.
+
+    In this module, the `weight` and `bias` are of :class:`torch.nn.Uninitialized`
```

Comment:
The class is UninitializedParameter, not Uninitialized.

Reply:
Instead of re-explaining what a lazy module is I think this can redirect the user to the lazy module documentation itself.

Reply:
updated